cfi_csrs.adoc

```diff
@@ -16,41 +16,32 @@ This chapter specifies the CSR state of the Zicfisslp extension.
   {bits:  1, name: 'CBZE'},
   {bits: 51, name: 'WPRI'},
   {bits:  1, name: 'SFCFIE'},
-  {bits:  1, name: 'CFIE'},
+  {bits:  1, name: 'SBCFIE'},
   {bits:  1, name: 'HADE'},
   {bits:  1, name: 'PBMTE'},
   {bits:  1, name: 'STCE'},
 ], config:{lanes: 4, hspace:1024}}
 ....
 
-The `CFIE` field (bit 60) controls whether Zicfisslp extension is available for
-use in modes less privileged than M. When `CFIE` is 1, the `SFCFIE` field
-(bit 59) activates forward-edge CFI at S-mode.
+Zicfiss extension introduces the `SBCFIE` field (bit 60) in `menvcfg`. When
+`SBCFIE` field is 1, the Zicfiss extension is active in S-mode. When `SBCFIE`
+field is 0, the Zicfiss extension is not active in S-mode and the following
+rules apply to privilege modes less than M.
```

Comment:
Machine mode should be aware of the machine. We have not generally defined resets for controls affecting lower privilege to support a unaware machine mode. The reset state of `mseccfg.MFCFIE` is not defined as it may be set to 0 or 1 depending on the platform. The `ELP` should be defined to reset to `NO_LP_EXPECTED`.